src/Components/WebAssembly/DebugProxy/src/Startup.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Net;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using WebAssembly.Net.Debugging;
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.DebugProxy
+{
+    public class Startup
+    {
+        public void Configure(IApplicationBuilder app, DebugProxyOptions debugProxyOptions)
+        {
+            app.UseDeveloperExceptionPage();
```

Comment:
`UseExceptionHandler`?

Reply:
I'm not sure why. The developer exception page provides useful diagnostic info, and this is a development-time feature only.